ffmpeg/ffmpeg.go

```diff
@@ -245,6 +245,8 @@ type MediaFormatInfo struct {
 	Acodec, Vcodec string
 	PixFormat      PixelFormat
 	Width, Height  int
+	FPS            float32
+	Dur            float32
```

Comment:
not sure that float32 is the best type for either of these, but if you don't care about precision it might be ok. i would opt for a rational for fps and int64 for (scaled) duration

Reply:
It's a good idea, I think float32 is sufficient precision for what we need right now